_includes/header.html

```diff
@@ -33,16 +33,17 @@
                 {% endfor %}
                 {% endif %}
             </div>
-            <a type="button" style="margin:auto 5px" class="btn btn-warning btn-sm" href="{{ site.baseurl }}/support">
+            <a type="button" style="margin:auto 5px; -webkit-appearance: none;" class="btn btn-warning btn-sm" href="{{ site.baseurl }}/support">
                 Support
             </a>
-            
-            <a type="button" style="margin:auto 5px" class="btn btn-warning btn-sm" href="{{ site.baseurl }}/join">
+
+            <a type="button" style="margin:auto 5px; -webkit-appearance: none;" class="btn btn-warning btn-sm" href="{{ site.baseurl }}/join">
                   Join
-            </a>    
-                  <a href="{{ site.baseurl }}/search" type="button" style="padding-left: 20px;">
-                    <i style="margin-top:15px; ;color:white; font-size:25px" class="fa fa-search"></i>
-                </a>
+            </a>
+
+            <a href="{{ site.baseurl }}/search" type="button" style="padding-left: 20px; -webkit-appearance: none;">
+                <i style="margin-top:15px; ;color:white; font-size:25px" class="fa fa-search"></i>
```

Comment:
Another little cleanup of an extra semi-colon.
```suggestion
                <i style="margin-top:15px; color:white; font-size:25px" class="fa fa-search"></i>
```